README.md

```diff
@@ -35,7 +35,12 @@ The stylesheets are installed in `$PREFIX/share/ocr-transform/xslt`.
 
```

Comment:
I don't understand this sentence. Is it important and related to the usage at all?